bin/nodenv-each

```diff
@@ -32,6 +36,12 @@ for arg in "$@"; do
     -g | --system )
       include_system=1
       ;;
+    -p | --parallel )
+      parallel=1;
+      ;;
+    -j=* | --jobs=* )
+      jobs="${arg#*=}"
```

Comment:
Question: what does `parallel` do if a user requests more jobs than `nproc` allows? Hopefully it just does the "nice" thing and proceeds with max jobs possible. And that would be great; I don't think we need any logic specific to that case unless parallel blows up in that scenario.

Reply:
`parallel` will run as many jobs as you tell it to use. There's nothing (aside from system resources) preventing say, 1,000 or 1,000,000 or however many you ask for.

Reading the man page a bit more I think specifying `0` as the default might be a good idea. It runs each job we want to run, in parallel, compared to running at most `nproc` jobs.